cloudify/utils.py

```diff
@@ -647,3 +648,88 @@ def get_admin_api_token():
 
 
 internal = Internal()
+
+
+def extract_plugins_to_install(plugin_list, filter_func):
+    """
```

Comment:
The docstring is too similar to the impl, it's needs to describe what part it's doing

Reply:
Changed